src/js/player.js

```diff
@@ -1158,6 +1158,49 @@ class Player extends Component {
     // Update the duration if available
     this.handleTechDurationChange_();
 
+    // if manual auto play is set to muted, any, or any truthy value
+    // we will try to auto play manually, rather than using the attribute
+    //
+    // - muted: mutes the media element and then calls play
+    // - truthy value: will just call play
+    // - any: Will call play and if the promise fails, it will try to mute
+    //        and then play
+    if (this.options_.manualAutoplay) {
```

Comment:
I still think that maybe `autoplay` is best and have these options:
- `muted` - will mute the video and then play
- `manual` - will call play manually
- `any` - (or maybe another name?) try to play unmuted and then mute if it fails
- any other value do current behavior

then the autoplay getter and setter can also handle this whenever it gets called and the autoplay getter can know whether this option was set and which option.

then this line will just turn to something like `this.autoplay(this.options_.autoplay)` or something.

Reply:
I realize that this overloads the values for the `autoplay` option, but it really is the best name and we can always deprecate the old values and go forward with just the new stuff.

Reply:
also, we could theoretically, have the setter return the play promise in that way if called later on

Reply:
OK we are going to go this route, let me re-work this pr